docs/2_customization/xml_to_es.md

```diff
@@ -215,14 +238,17 @@ Datura's `lib/datura/to_es/tei_to_es/fields.rb` has a list of functions which ge
 The following are helpers created to make your life easier:
 
 - get_text
+  - grab the text of one or many xpaths
 - get_list
+  - create a list of text from one or many xpaths
```

Comment:
After our discussion yesterday, it sounds like maybe with better documentation and a few examples this can remain `get_list`, but we'll see what you think after I add them!

Reply:
About 12 lines down from the lines you commented there is an example comparing the output of get_text and get_list.  Should I add more?

Reply:
D'oh. Sorry. I thought I had expanded the nearby lines to look for such an example. I think this is fine. I probably won't forget the distinction after having discussed it more too :)